### DISCUSSION TITLE
日志里高频出现连接失败错误

### DISCUSSION THREAD

**Discussion by neopu8** [jeessy2/ddns-go#345] [General]
日志高频率出现“ 连接失败! [点击查看接口能否返回IPv4地址](https://ip.3322.net/)”，每次点击链接后浏览器都有返回IP地址。
增加了其他地址也是类似出现连接失败的错误。

会不会是建立连接的时间太短了？我应该怎么设定和优化好一些？

PS.
v4.0.1

**Comment by neopu8** [comment]
再补充一下：
用浏览器打开连接，总是需要时间处理才返回IP地址的，有时返回得很快，有时很慢，甚至会超过10s。
是否能设定个等待返回的响应时间，这样可以提高返回IP的成功率啊？

超级喜欢这个产品，希望越来越好。

**Comment by jeessy2** [comment]
更新频率设置的多少

**Comment by neopu8** [comment]
5分钟一次。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jeessy2/ddns-go"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月3日(星期三) 上午8:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jeessy2/ddns-go] 日志里高频出现连接失败错误 (Discussion #345)





 
更新频率设置的多少

**Reply by jeessy2** [reply]
贴个日志看下

**Reply by jeessy2** [reply]
试下v4.0.3版本

**Comment by yjwskj** [comment]
老大！能不能加入https://dynv6.com项目

**Comment by neopu8** [comment]
前几天出差了没回复。
今天试了v4.0.4版本，观察看看。
很感谢大佬的支持，非常非常喜欢这个咚咚，希望越来越好！



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: jeessy2 ***@***.***&gt;
发送时间: 2022年8月4日 19:06
收件人: jeessy2/ddns-go ***@***.***&gt;
抄送: neopu8 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [jeessy2/ddns-go] 日志里高频出现连接失败错误 (Discussion #345)






试下v4.0.3版本

**Comment by zhimibuwudn** [comment]
4.0.4我这也是经常提示这个错误获取好多次才能正确配置地址
2022/08/30 07:35:50 连接失败! [点击查看接口能否返回IPv4地址](https://myip4.ipip.net/),
2022/08/30 07:36:20 连接失败! [点击查看接口能否返回IPv4地址](https://ddns.oray.com/checkip),
2022/08/30 07:36:50 连接失败! [点击查看接口能否返回IPv4地址](https://ip.3322.net/),
2022/08/30 07:36:50 未能获取IPv4地址, 将不会更新
这几个接口都可以正常获取到地址的

**Reply by jeessy2** [reply]
你的更新频率多少？
把正常和不正常的日志发来看看，完整的

**Reply by zhimibuwudn** [reply]
默认的五分钟linux默认安装的日志路径在哪

**Reply by jeessy2** [reply]
服务安装在 /var/log/ 下，有个ddns-go开头的日志文件
